app/version.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Maybe put some placeholder values in here so that the footer makes a bit more sense when in development.

Reply:
but it should be taken off config, so better to add a default value to the config

Reply:
@tamlyn good idea. @g-sam bad idea :)